coalib/bears/Bear.py

```diff
@@ -316,9 +316,9 @@ def setup_dependencies():
         """
 
     @classmethod
-    def check_prerequisites(cls):
+    def check_all_deps(cls):
```

Comment:
I am not sure but isn't this kind of risky? since there are other bears in coala-bears which are implementing this method with check_prequisities and later tests call that. this requires to have those changes in coala-bears as well, right?